sql/sql_parse.cc

```diff
@@ -1437,7 +1437,7 @@ bool dispatch_command(THD *thd, const COM_DATA *com_data,
       else
       {
         mysql_reset_thd_for_next_command(thd);
-        my_error(ER_LOCK_DEADLOCK, MYF(0), "wsrep aborted transaction");
+        my_message(ER_LOCK_DEADLOCK, "WSREP cancelled the transaction", MYF(0));
```

Comment:
We could do this, there are several types of errors that all use ER_LOCK_DEADLOCK, so it would be nice to split these off.  We would need to determine an appropriate range to avoid future conflicts (maybe start from 10000 or 20000).